themes/default/languages/english/Admin.english.php

```diff
@@ -129,9 +129,9 @@
 $txt['attachment_delete_admin'] = '[attachment deleted by admin]';
 $txt['live'] = 'Latest Software Updates';
 $txt['remove_all'] = 'Clear Log';
-$txt['agreement_not_writable'] = 'Warning - agreement.txt is not writable, any changes you make will NOT be saved.';
+$txt['agreement_not_writable'] = 'Warning - agreement.txt is not writable any changes you make will NOT be saved.';
```

Comment:
Oh no, the comma should be replaced by a period instead. ;-)

Reply:
Or you could insert a "so" but I think two smaller sentences is probably better stylistically.